sched/job_def_serializer_test.go

```diff
@@ -1,3 +1,5 @@
+// +build property
```

Comment:
On second thought, what do we gain by distinguishing between unit/property tests? Anything at all?